jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/Checkbox.java

```diff
@@ -3,16 +3,26 @@
 import com.epam.jdi.light.asserts.generic.HasAssert;
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
 import com.epam.jdi.light.elements.interfaces.base.HasCheck;
 import com.epam.jdi.light.elements.interfaces.base.HasClick;
 import com.epam.jdi.light.material.asserts.inputs.CheckboxAssert;
 
 public class Checkbox extends UIBaseElement<CheckboxAssert> implements HasClick, HasCheck,
```

Comment:
Do you really need part with implements?

Reply:
Really not